repoarchivetool/app.py

```diff
@@ -606,6 +628,58 @@ def confirm_action():
 
     return flask.redirect("/")
 
+@app.route('/insert_test_data', methods=['POST', 'GET'])
```

Comment:
Ideally we would only expose this route if the test_data feature were configured using feature flags.  This allows us to only enable the feature on dev platforms and disable the feature on prod platforms meaning we can have the same code on all environments.

Since we are doing this to a tight timescale and what you have is fit for the demo purpose, I am happy if you want to check in as is but I think we might want to add a ticket to enable configurable optioning of the code.

Reply:
Added ticket for future development